package.json

```diff
@@ -81,6 +81,11 @@
   "ember": {
     "edition": "octane"
   },
+
+  "dependencies": {
+    "msw": "^0.47.0",
+    "sinon": "^14.0.0"
+
```

Comment:
Closing brace missing

Reply:
no in the file I can't see that 

![image](https://user-images.githubusercontent.com/61110378/190840975-548b3eb4-dac8-4cc7-9a84-356d46a5fbfc.png)